src/util/types.ts

```diff
@@ -101,11 +101,12 @@ export interface AdapterMetricsMeta {
 }
 
 /**
- * Map of overrides objects (symbol -\> symbol) per adapter name
+ * Map of overrides objects (symbol -\> symbol and request transport) per adapter name
  */
 export type Overrides = {
   [adapterName: string]: {
     [symbol: string]: string
+    __TRANSPORT__: string
```

Comment:
Alternatively we could introduce a new top level object to hold these in, to guarantee no collisions - maybe something like `adapterOverrides`? It's a shame that `overrides` is already being used just for symbols - I think the likelihood of `transport` colliding with any `symbols` is low enough that I'd be tempted to keep using `overrides`.

Reply:
Agree. I also think that the collision probability is very low, the underscore syntax was used for readability mostly, to easily spot the transport in the RDD if there are many other overrides. Regarding having that in another top level property i think it would be counter intuitive as 'overrides' is the actual place for that. Furthermore, the 'transport' is the only new thing that can be overridden. Apart from custom input params we have 3 common input params, 
`endpoint`, `transport` and `overrides`.  We cannot override the `endpoint` and `overrides` properties in the overrides, but only the transport. Thought it's not a good idea to have separate  top lvl object just for transport. 
With that said i'll change the `__TRANSPORT__` to `transport`.